Research CSX File not finding PythonNet

<!--- This template provides sections for bugs and features. Please delete any irrelevant sections before submitting -->

#### Expected Behavior
<!--- Required. Describe the behavior you expect to see for your case. -->
CSX File finds PythonNet dll.

#### Actual Behavior
<!--- Required. Describe the actual behavior for your case. -->

Currently not finding it because the location has changed since QC PythonNet 2.0.0 release.

```
error CS0006: Metadata file `pythonnet/Python.Runtime.dll' could not be found
```

#### Potential Solution
<!--- Optional. Describe any potential solutions and/or thoughts as to what may be causing the difference between expected and actual behavior. -->

Replace .dll directive, with new location, which is just in the bin.

```
#r "pythonnet/Python.Runtime.dll"
```

to

```
#r "Python.Runtime.dll"
```


#### Reproducing the Problem
<!--- Required for Bugs. Describe how to reproduce the problem. This can be via a failing unit test or a simplified algorithm that reliably demonstrates this issue.  -->
Run research and load in QuantConnect.csx

#### System Information
<!--- Required for Bugs. Include any system specific information, such as OS. -->
Applicable to research env on all OS's

#### Checklist
<!--- Confirm that you've provided all the required information. -->
<!--- Required fields --->
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
<!--- Required for Bugs, feature request can delete the line below. -->
- [x] I have provided detailed steps to reproduce the issue

<!--- Template inspired by https://github.com/stevemao/github-issue-templates -->

--- a/Research/QuantConnect.csx
+++ b/Research/QuantConnect.csx
@@ -1,4 +1,4 @@
-﻿#r "pythonnet/Python.Runtime.dll"
+﻿#r "Python.Runtime.dll"
 #r "QuantConnect.Algorithm.dll"
 #r "QuantConnect.Algorithm.Framework.dll"
 #r "QuantConnect.Common.dll"

